kubectl-plugin/src/main.rs

```diff
@@ -21,12 +21,15 @@ use structopt::StructOpt;
 use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt, Registry};
 use yaml_rust::YamlLoader;
 
+extern crate utils as mayastor_utils;
+
 /// OpenTelemetry helpers for handling Processor Tags
 pub mod opentelemetry_helper {
     include!("../../common/src/opentelemetry.rs");
 }
 
 #[derive(StructOpt, Debug)]
+#[structopt(version = mayastor_utils::package_info!(mayastor_utils::git_version()))]
```

Comment:
Should we add this to the other binaries?

Reply:
Done